src/EnergyPlus/ZonePlenum.cc

```diff
@@ -881,6 +891,16 @@ namespace ZonePlenum {
 				Node( ZoneNodeNum ).HumRat = OutHumRat;
 				Node( ZoneNodeNum ).Enthalpy = PsyHFnTdbW( Node( ZoneNodeNum ).Temp, Node( ZoneNodeNum ).HumRat );
 
+				ZoneSupPlenCond( PlenumZoneNum ).ZoneTemp = 20.0;
+				ZoneSupPlenCond( PlenumZoneNum ).ZoneHumRat = 0.0;
+				ZoneSupPlenCond( PlenumZoneNum ).ZoneEnthalpy = 0.0;
+				ZoneSupPlenCond( PlenumZoneNum ).InletTemp = 0.0;
+				ZoneSupPlenCond( PlenumZoneNum ).InletHumRat = 0.0;
+				ZoneSupPlenCond( PlenumZoneNum ).InletEnthalpy = 0.0;
+				ZoneSupPlenCond( PlenumZoneNum ).InletPressure = 0.0;
+				ZoneSupPlenCond( PlenumZoneNum ).InletMassFlowRate = 0.0;
+				ZoneSupPlenCond( PlenumZoneNum ).InletMassFlowRateMaxAvail = 0.0;
+				ZoneSupPlenCond( PlenumZoneNum ).InletMassFlowRateMinAvail = 0.0;
```

Comment:
So here each plenum node data is reset individually each BeginEnvirnFlag. Would it be smarter to do the first reset EACH  BeginEnvrnFlag so that all plenums are reset at the same time in case something I can't think of needs all plenums to be reset at the same time? e.g.,

    if ( ! BeginEnvrnFlag ) {
        InitAirZoneReturnPlenumEnvrnFlag  = true;
     }

Reply:
My intent was to reset all plenums once at the beginning of each environment.   Will recheck that.

Reply:
This is just weird. At the top of this loop the node data is reset. That's what ResetNodeData is supposed to do, except the humrat probably isn't reset to outdoor humrat.

No show stoppers here.

Reply:
ResetNodeData sets everything to 0 so I wonder why this needs a special case?

Reply:
@rraustad Looking at the whole init functions, for both supply and return plenums, the begin environment init happens once for each environment for all plenums.  If these were OO'd we'd initialize each one individually, but this should be OK for now.